homeassistant/components/synology_dsm/const.py

```diff
@@ -8,6 +8,7 @@
 DOMAIN = "synology_dsm"
 
 CONF_VOLUMES = "volumes"
+BASE_NAME = "Synology"
 DEFAULT_NAME = "Synology DSM"
```

Comment:
Not used anymore

Reply:
Still used in `sensor.py` line 78

Reply:
`BASE_NAME` is used line 78, not `DEFAULT_NAME` :wink: 

```suggestion
```

Reply:
Looks like commit suggestion didn't take but you got it anyways :)